docs/proposals/container-init.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
```

Comment:
This change was autogenerated after running pre-commit hook. Not sure why it's appearing.